README.md

```diff
@@ -37,11 +37,13 @@ cargo run --example hello-compute 1 2 3 4
 
 #### Run Examples on the Web (`wasm32-unknown-unknown`)
 
-Running on the web is still work-in-progress. You may need to enable experimental flags on your browser. Check browser implementation status on [webgpu.io](https://webgpu.io).
+Running on the web is still work-in-progress. You may need to enable experimental flags on your browser. Check browser implementation status on [webgpu.io](https://webgpu.io). Notably, `wgpu-rs` is often ahead in catching up with upstream WebGPU API changes. We keep the `gecko` branch pointing to the code that should work on latest Firefox.
 
 To run examples on the `wasm32-unknown-unknown` target, first build the example as usual, then run `wasm-bindgen`:
 
 ```bash
+# Checkout `gecko` branch that matches the state of Firefox
+git checkout upstream/gecko
```

Comment:
can we make this origin instead of upstream.
Thats what the majority of people will have.

Reply:
Sorry, I forgot to address this!
Why do you think `origin` would be proper here?
Typically it points to your fork of wgpu-rs, not to our repo. And who knows what branches your fork has?

Reply:
Well normally people clone the main repo first and then add their fork as a second remote when they need to contribute.

This is because:
I want to run the examples so let's clone the repo.
Happens before:
I want to submit a patch so let's fork it on GitHub.

Reply:
I can second this "origin" is gfx, "cwfitzgerald" is mine.

Reply:
Not totally surprised I use it differently :)
Ok, let's change this line to origin/gecko